src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -252,9 +259,11 @@ MulticopterAttitudeControl::vehicle_status_poll()
 		if (_actuators_id == nullptr) {
 			if (_vehicle_status.is_vtol) {
 				_actuators_id = ORB_ID(actuator_controls_virtual_mc);
+				_attitude_sp_id = ORB_ID(mc_virtual_attitude_setpoint);
```

Comment:
This is inside the check for `_actuators_id == nullptr`. That's not intended, right?

Reply:
@julianoes It is. They both need to be set but we only use one of them to do the check.

Reply:
Ok, I guess it's just confusing for the reader then.